app/views/unattended/provisioning_templates/snippet/puppet.conf.erb

```diff
@@ -13,7 +13,7 @@ description: |
   os_name   = @host.operatingsystem.name
 
   aio_enabled = host_param_true?('enable-puppetlabs-repo') ||  host_param_true?('enable-official-puppet8-repo') || host_param_true?('enable-official-puppet7-repo') || host_param_true?('enable-puppet7') || host_param_true?('enable-puppetlabs-puppet6-repo') || host_param_true?('enable-puppet6') || host_param_true?('enable-puppetlabs-puppet5-repo') || host_param_true?('enable-puppet5')
-  aio_available = os_family == 'Debian' || os_family == 'Redhat' || os_name == 'SLES'
+  aio_available = os_family == 'Debian' || os_family == 'Redhat' || os_family == 'Suse'
```

Comment:
```console
# foreman-rake console
irb(main):001:0> os = Operatingsystem.find { |os| os.name == 'openSUSE' }
=> #<Suse id: 41, major: "15", name: "openSUSE", minor: "5", nameindicator: nil, created_at: "2023-07-14 18:16:29.812119000 +0000", updated_at: "2023-07-14 18:16:29.812119000 +0000", release_name: "", type: "Suse", description: "openSUSE Leap 15.5", password_hash: [FILTERED], title: "openSUSE Leap 15.5">
irb(main):002:0> os.title
=> "openSUSE Leap 15.5"
irb(main):003:0> os.family
=> "Suse"
```

Which matches the OS fact on openSUSE;
```console
silver:~ # facter os.name
openSUSE
silver:~ # facter os.family
Suse
```

Reply:
Nice. this is interesting because:

https://github.com/theforeman/foreman/blob/develop/app/models/operatingsystems/suse.rb#L17


and

![image](https://github.com/theforeman/foreman/assets/25485845/3fa0f7e6-7366-4224-bdc4-ced910ca8e97)

Reply:
Ah, capitalization, always a fun time.

I guess I could throw a downcase in there, and do the OS family comparisons case-insensitive instead, then it won't look as confusing

Reply:
Thanks for the info. Then I think we are good to merge this.